app/src/models/data/IReport.ts

```diff
@@ -2,22 +2,16 @@
 export interface IReport {
     /** Unique report ID */
     reportId: string;
-    contactInformation: {
```

Comment:
Is there a reason to flatten the contactsInformation, issueLocation, etc? Either way is fine, but it kind of makes sense to group certain values